# Define the goal
string desired_object_label # Object Detection Class Name
uint32 mode # See COMMON_NAMES::NAV_VISION_TYPE, two modes, follow and reach
---
# Define the result
# See COMMON_NAMES::NAV_VISION_RESULT for possible values
uint32 result
---
# Define a feedback message
float32 percent_complete